packages/base-styles/_variables.scss

```diff
@@ -16,6 +16,12 @@ $editor-line-height: 1.8;
 $big-font-size: 18px;
 $mobile-text-min-font-size: 16px; // Any font size below 16px will cause Mobile Safari to "zoom in"
 $border-width: 1px;
+$border-width-focus: 1.5px;
+$border-width-tab: 4px;
+
+:root {
```

Comment:
Is there a better place than this SCSS file to output these? I imagine there is, because right now the variable appears duplicated a bunch of times, and it shouldn't be. I would appreciate advice here.

Reply:
What's the reasoning for using a CSS variable here. It seems inconsistent with the other variables (gris, colors...)

Reply:
They're both variables, that's all the reasoning there is. And that's why I was asking for advice on what might be a better place. It also gets duplicated when here, so it really isn't the best place — just not sure where to put it.

Reply:
I think the best way to do it would be something like

```
$border-width-focus: var( --border-width-focus, 1.5px );
```

and just use `$border-width-focus` everywhere without this root rule.

That said, I'd prefer if we avoid introducing these variables in this PR in order to give it more thoughts and also be consistent with the variables we propose.

Reply:
The reason for doing it the way it's been done currently, is to provide a fallback for IE11.

Reply:
It can potentially be automated using a Post CSS plugin. Maybe by replacing `var` above with a custom one `config` or something

Reply:
Good argument for leaving it to a separate PR.